src/lifecycle.js

```diff
@@ -37,12 +38,12 @@ function ensureLifecycleFlag (target, component, name) {
 }
 
 /**
- * Parses an event definition and returns information about it.
- *
- * @param {String} e The event to parse.
- *
- * @returns {Object]}
- */
+* Parses an event definition and returns information about it.
+*
+* @param {String} e The event to parse.
+*
+* @returns {Object]}
+*/
```

Comment:
Wait, is this style of doc comment `*` intentional?

It looks like this:

```text
/**
*
*/
```

when every bit of code that does the `/**` thing looks like this:

```text
/**
 *
 */
```